chain-api/src/utils/dex/position.helper.ts

```diff
@@ -32,28 +32,27 @@ import { requirePosititve } from "./format.helper";
  */
 export function updatePositions(
   positions: Positions,
-  owner: string,
+  nftId: string,
```

Comment:
```suggestion
  nftId: TokenInstanceKey,
```
(or an appropriately modeled key object that also contains `collection`, `category`, `type`, `additionalKey`, and `instance` properties)